### PR TITLE
Added Arastta eCommerce

### PR DESCRIPTION
**Pull request by burakcakirel**

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -431,6 +431,16 @@
 			"script": "adnxs\\.(?:net|com)",
 			"website": "appnexus.com"
 		},
+        "Arastta": {
+          "cats": [
+            6
+          ],
+          "excludes": "OpenCart",
+          "script": "ar\\.js",
```
Comment (AliasIO) [inline src/apps.json]: Is it possible to make this more specific? It will match any JavaScript files that ends with 'ar'.

File: src/apps.json [hunk 2]
```diff
@@ -431,6 +431,16 @@
...
+          "html": "Arastta",
```
Comment (AliasIO) [inline src/apps.json]: This will match any page that has the word 'Arastta' on it. HTML patterns should at least contain `<` or `>`.

**Comment by kolen** [comment]
Maybe can be detected with:

- ``<input type="hidden" name="redirect" value="path=0_59&route=product/category" />`` (with numbers/names replaced with ``\d+``, etc)
- ``<p> .* Powered by <a href="http://arastta.org">Arastta eCommerce</a></p>``

**Comment by q--** [comment]
`.*` is undesirable here: it will find *any* `<p>` preceding the rest of the code anywhere in the document.

**Comment by burakcakirel** [comment]
Will you merge this PR ?

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**